tools/taskcluster/decision_task.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import json
+import os
+
+import taskcluster
+
+from tasks import schedule_task_graph, TaskBuilder
+
+
+def pr(builder, branch):
```

Comment:
Fun fact, I'm moving `builder` to a parameter, for the taskgraph conversion too!

Reply:
Yeah, it can arguably be global, but that's only because `TASK_ID`, `MOBILE_HEAD_REPOSITORY` and `MOBILE_HEAD_REV` are global environment variables, which is :unamused:. It's interesting that `taskgraph` requires it to be a parameter, though :thinking: